### ISSUE TITLE
Correct settings for modpacks

### ISSUE THREAD

**Issue by suoko** [PojavLauncherTeam/pojavlauncherteam.github.io#215]
Is anyone collecting the necessary settings to run modpacks ?
 I've been playing with  various devices (android with mediatek and snapdragon SOC, chromeos on arm and fydeos on Intel) and they all need different settings regarding renderer, Java tweks, etc...
I'm doing my tests using a light modpack like "lightly sprinkled".
A database collecting those settings would save some time, or maybe they can be applied automatically when pojav gets installed

**Comment by alexytomi** [comment]
No, it seemed like it would be an enormous amount of work for us to even try doing.

I guess this could be turned into a feature request for community submitted settings but that'd also be pretty hard since we would need to have the same phone (including specification between region variants), same software (Software Updates can break certain renderers on specific things), and maintaining that list for newer pack versions.

It seemed it'll be too complex IMHO. Perhaps in future android land where most devices would have the same behavior instead of barfing themselves for reasons we cannot even attempt to troubleshoot.

**Comment by RadiatedExodus** [comment]
Unfortunately as @alexytomi said, maintaining such a list would be too volatile and requires a lot of maintenance. It's better to have users try modpacks on their own and see if it works on their device on whatever configuration they are using.
